### ISSUE TITLE
viewer: prevent or cancel thumbnail download when scrolling long lists fast

### ISSUE THREAD

**Issue by OzoneGrif** [pacien/ldgallery#200]
On long thumbnail lists, if you scroll fast (ie. because you know the picture is near the end), it will trigger all the intermediate thumbnails.

It could be a good idea to either wait a few milliseconds before starting the download, and/or cancel a download which barely started.

You can cancel a download by setting the img source to an empty string.

**Comment by pacien** [comment]
Agreed: this is especially needed when the gallery is served from a web server which enforces some kind of rate-limiting.
